### PR TITLE
Start displaying cards

### PR DESCRIPTION
**Pull request by Tarig0**

### CODE REVIEW

File: src/app/shared/models/icon.enum.ts [hunk 1]
```diff
@@ -4,9 +4,11 @@
 export enum Icon {
     Quiddity1,
     Quiddity2,
-    Quiddity3,
     Draw1,
     Draw2,
     ReRoll,
-    Ready
+    Ready,
+    Quiddity3OrReRoll,
```
Comment (Tarig0) [inline src/app/shared/models/icon.enum.ts:11]: @phyxl would you rather these three compound icons be treated as their own value or be added to the dieside as an array?

File: src/app/shared/models/type.ts [hunk 2]
```diff
@@ -1,11 +1,73 @@
...
 /**
  * Holds the data about a Card
 */
 export class Type {
-    class: Class;
-    name: string;
-    description: string;
-    cost: number;
-    glory: number;
+    private intAbilities: string[];
```
Comment (Tarig0) [inline src/app/shared/models/type.ts:7]: To be replaced by a class with the description of the ability stored within

File: src/app/card/card.component.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { Type } from '../shared/models/type';
+
+@Component({
+  selector: 'app-card',
+  templateUrl: './card.component.html',
```
Comment (Tarig0) [inline src/app/card/card.component.ts:6]: Todo:: actually style the card

File: src/app/shared/models/icon.enum.ts (hunk 1 above)
Comment (phyxl) [reply src/app/shared/models/icon.enum.ts:11]: It's best as an array I think, to avoid repeating ourselves

Comment (Tarig0) [reply src/app/shared/models/icon.enum.ts:11]: we'll need a bool to decide whether the array is or/and

**Pull request merged**